src/seedu/addressbook/data/person/Name.java

```diff
@@ -6,26 +6,28 @@
 import java.util.List;
 
 /**
- * Represents a Person's name in the address book.
- * Guarantees: immutable; is valid as declared in {@link #isValidName(String)}
+ * Represents a Person's name in the address book. Guarantees: immutable; is
+ * valid as declared in {@link #isValidName(String)}
  */
 public class Name {
-
+    
     public static final String EXAMPLE = "John Doe";
-    public static final String MESSAGE_NAME_CONSTRAINTS = "Person names should be spaces or alphabetic characters";
-    public static final String NAME_VALIDATION_REGEX = "^[\\p{L}0-9 .'-]+$";  
+    public static final String MESSAGE_NAME_CONSTRAINTS = "Person names should consist of unicode letters, spaces, numbers, ', - or .";
```

Comment:
This message is user-facing, so probably remove the `unicode` since that's a technical term that users don't need to know about.